plugin/ochttp/httptrace_test.go

```diff
@@ -139,7 +139,7 @@ func TestHandler(t *testing.T) {
 		t.Run(tt.header, func(t *testing.T) {
 			propagator := &testPropagator{}
 
-			handler := NewHandler(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Please see my suggestion to keep `NewHandler(http.Handler)`

Reply:
Done